portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,73 @@
+body {
+  font-family: "Sanchez", Verdana, Arial, Helvetica, sans-serif;
+  font-size: 15px;
+  background: url("/images/moroccan-flower-dark.png"), #6F4A38;
```

Comment:
Recommend to use only lowercase: https://google.github.io/styleguide/htmlcssguide.html#Capitalization